example-project/webpack.config.js

```diff
@@ -15,7 +15,7 @@ const { resolve, join } = require('path');
 
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
-const CleanWebpackPlugin = require('clean-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
```

Comment:
Greenkeeper might have no chill, but when checking up on it, this wasn't working anyway...so there's that.